docs/telegram-usage.md

```diff
@@ -87,6 +87,35 @@ Example configuration showing the different settings:
    },
 ```
 
+## Create a custom keyboard (command shortcut buttons)
+Telegram allows us to create a custom keyboard with buttons for commands.
+The default custom keyboard looks like this.
+```python
+[
+    ['/daily', '/profit', '/balance'], # row 1, 3 commands
+    ['/status', '/status table', '/performance'], # row 2, 3 commands
+    ['/count', '/start', '/stop', '/help'] # row 3, 4 commands
+]
+``` 
+### Usage
+You can create your own keyboard in `config.json`:
+``` json
+"telegram": {
+      "enabled": true,
+      "token": "your_telegram_token",
+      "chat_id": "your_telegram_chat_id",
+      "keyboard": [   
+          ["/daily", "/stats", "/balance", "/profit"],
+          ["/status table", "/performance"],
+          ["/reload_config", "/count",   "/logs"]
+      ]
+   },
+```
+!!! Note    Only a certain list of commands are allowed. Command arguments are not
+supported!
```

Comment:
I can only recommend to use `mkdocs serve` to see how this will look like (and to ensure  it renders correctly) (taken from here: https://www.freqtrade.io/en/latest/developer/#documentation)

```suggestion
!!! Note
    Only a certain list of commands are allowed. Command arguments are not
supported!
```

Reply:
Yes thank you! Yes, I need those pre commit hooks, you are right! :+1: